bolero-generator/Cargo.toml

```diff
@@ -16,6 +16,7 @@ std = ["alloc", "either/use_std"]
 alloc = ["rand_core/alloc"]
 
 [dependencies]
+arbitrary = "1.2"
```

Comment:
Let's make this optional and opt-in. I'd prefer to keep the MSRV relatively conservative.

I think I'd also prefer to use `1` so we're not overly restrictive on downstream crates.